contracts/src/CrossChainApplications/examples/NativeTokenBridge/NativeTokenDestination.sol

```diff
@@ -41,12 +40,10 @@ contract NativeTokenDestination is TeleporterOwnerUpgradeable, INativeTokenDesti
     address public constant BURNED_TX_FEES_ADDRESS = 0x0100000000000000000000000000000000000000;
 
     /**
-     * @notice Designated Blackhole Address for this contract.
-     *
-     * @dev Tokens are sent here to be "burned" before sending an unlock message to the source chain. Different
-     * from the burned tx fee address so they can be tracked separately.
+     * @dev Used when reporting burned gas fees to the source chain. Fees will be unlocked and sent to this address.
+     * Different from the burned tx fee address so they can be tracked separately.
      */
-    address public constant BURN_FOR_TRANSFER_ADDRESS = 0x0100000000000000000000000000000000010203;
+    address public constant GENERAL_BURN_ADDRESS = 0x0100000000000000000000000000000000010203;
```

Comment:
nit: I prefer the old name, `TRANSFER_TO_SOURCE_BURN_ADDRESS` or similar. Also, can the comment above be updated.

Reply:
We use it on both chains. On the destination chain as well to burn native tokens before transferring to the source chain.

Reply:
I think it would be clearer if they were two separate variables, even if they contained the same address (I don't have a strong preference on if they should or not)